.travis.yml

```diff
@@ -12,6 +12,7 @@ before_script:
   - composer install --no-interaction --prefer-source --dev
   - ~/.nvm/nvm.sh install v0.6.14
   - ~/.nvm/nvm.sh run v0.6.14
+  - 'if [ "$TRAVIS_PHP_VERSION" == "7.0" ]; then echo "xdebug.overload_var_dump = 1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini fi'
```

Comment:
I wrote that I don't know why it failed, because I got a blank log in Travis yesterday.